cmd/crossplane/core/core.go

```diff
@@ -60,6 +62,10 @@ func (c *Command) Run(log logging.Logger) error {
 		return errors.Wrap(err, "Cannot add core Crossplane APIs to scheme")
 	}
 
+	if err := oamapis.AddToScheme(mgr.GetScheme()); err != nil {
+		return errors.Wrap(err, "Cannot add core OAM APIs to scheme")
+	}
+
 	if err := oam.Setup(mgr, log); err != nil {
```

Comment:
Should we just call the oam-kubernetes-runtime setup function and get rid of our own?